File: abc/abc128/a.java
// https://atcoder.jp/contests/abc128/submissions/41617793

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int p = stdIn.nextInt();
    p += a * 3;

    System.out.println(p / 2);
  }
}

File: abc/abc128/a.cs
// https://atcoder.jp/contests/abc128/submissions/46699958

class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    int p = int.Parse(t[1]);

    p += a * 3;
    Console.Write(p / 2);
  }
}

